coral/src/app/features/topics/details/subscriptions/TopicSubscriptions.tsx

```diff
@@ -43,8 +40,9 @@ const isSubscriptionsOption = (value: string): value is SubscriptionOptions => {
 
 const TopicSubscriptions = () => {
   const navigate = useNavigate();
-  // @ TODO get environment from useTopicDetails too when it is implemented2
-  const { topicName } = useTopicDetails();
+  const { topicOverview, environmentId } = useTopicDetails();
```

Comment:
How would refetching the `topicOverview` data work, for when we do actions that would need such a refresh? For example:
- on subscription tab
- send a Delete request for a subscription
- [behaviour not yet implemented, but will be] `showDeleteAcl` is now `false` on this subscription, so we need to refetch to update the Table with a disabled Delete button

Reply:
Yes, we could do a `queryClient.invalidateQuery` or refreshQuery when we need it!